layouts/partials/tw/nav.html

```diff
@@ -1,137 +1,311 @@
+  <ul
+    class="
+      -top-2 right-1 z-10 ml-1 box-content w-0 flex-wrap gap-2 overflow-hidden bg-white shadow sm:relative sm:top-auto sm:flex sm:h-auto sm:w-auto sm:max-w-4xl sm:flex-1 sm:flex-nowrap sm:items-center sm:justify-evenly sm:rounded-none sm:border-none sm:bg-transparent sm:bg-none sm:px-16 sm:shadow-none sm:before:content-[] md:overflow-visible md:px-[6.75rem]
+    "
+    :class="toggle ? 'max-sm:absolute max-sm:border rounded border-s-5 flex w-auto h-auto px-4 pt-14 pb-10 sm:pt-0 sm:pb-0 sm:overflow-visible' : ''"
+    x-on="toggle"
+    :id="$id('dropdown-button')"
+  >
+    {{ range site.Menus.main }}
+      <li
+        class="flex h-11 flex-[1_0_45%] items-center text-sm leading-none sm:flex-[0_0_auto] md:justify-center"
+      >
         <a
-          href="/search/"
-          class="site-nav:item"
-          data-ga-category="search"
-          :data-ga-action="'modal:search:open'"
-          x-data
-          @click.prevent="$dispatch('modal:search:open')"
+          class="flex-[0_0_auto] font-bold uppercase hover:underline hover:underline-offset-2 sm:font-normal"
+          href="{{ .URL }}"
+          data-ga-category="item"
+          >{{ .Name }}</a
         >
-          <svg class="site-nav:svg-icon" aria-label="Search">
-            {{ partial "helper/svg.html" "search" }}
-          </svg>
-        </a>
+      </li>
+    {{ end }}
+    {{ range site.Menus.main_about }}
+      <li
+        class="
+          mt-4 flex-[0_0_100%] flex-wrap items-center text-sm leading-none 
+          sm:absolute sm:z-10 sm:h-11 sm:w-0 sm:overflow-hidden 
+          md:right-1 md:top-0 md:my-0 md:flex
+          md:flex-[0_0_auto] md:overflow-visible
+        "
+        :class="toggle ? 'sm:block sm:right-[-4rem] md:right-[-4rem] sm:-top-6 sm:pt-2 sm:pb-3 sm:pr-8 sm:pl-4 sm:rounded-tr sm:bg-white sm:overflow-visible sm:w-48 sm:h-[auto] md:absolute md:h-auto md:top-[-1rem] md:pt-[15px] box-content sm:border-s-5 sm:border-r sm:border-t' : 'sm:overflow-hidden sm:w-0 sm:h-11'"
+      >
         <a
-          class="site-nav:donate-button"
-          href="/donate/"
-          data-ga-category="donate"
+          class="
+            force-justify-end sm:h block w-full flex-1 border-b py-4 font-bold uppercase hover:underline hover:underline-offset-2 sm:flex sm:w-full sm:items-center sm:capitalize md:py-0
+          "
+          :class="toggle ? 'sm:py-0 sm:h-11 md:justify-start' : 'md:justify-end'"
+          href="{{ .URL }}"
+          data-ga-category="item"
+          >{{ .Name }}</a
         >
-          <svg
-            class="site-nav:svg-icon"
-            viewBox="0 0 20 20"
-            aria-hidden="true"
-            style="top: 1px; margin-right: 2px"
+        {{ if .HasChildren }}
+          <ul
+            class="
+              my-3 flex w-full flex-wrap gap-2 md:h-0 md:w-0 md:overflow-hidden
+            "
+            :class="toggle ? 'md:visible md:overflow-visible md:w-auto md:h-auto sm:w-48' : 'md:hidden md:w-0'"
```

Comment:
It's probably worth using the `:class` object syntax here too, because if md:visible ever stops overriding md:hidden for some reason, this would break.

Reply:
I don't follow. I'm already using the `:class` syntax here, aren't I?

Reply:
never mind. I'm getting your comments out of order